packages/storybook-vue/stories/3_components/button/Button.stories.mdx

```diff
@@ -263,9 +263,8 @@ For Shadow Parts, please inspect the element's #shadow.
     <Story
         name='Link Download'
         args={{
-            href: 'assets/aa.png',
+            href: 'http://localhost:6006/assets/Telekom-Logo-Magenta-350x150.svg',
```

Comment:
ah damnit, thought that `aa.png` was just a temporary demo, but no 😅
maybe just put `assets/welcome.png`
(localhost isn't really available on the hosted instance 🙃)

Reply:
ok, I changed the download source ;-)